docker-entrypoint.sh

```diff
@@ -12,9 +12,16 @@ setup_database()
 
 run_data_migrations()
 {
-  echo "Running data migrations…"
-  bundle exec rake data:migrate
-  echo "Finished running data migrations."
+  if [ "$DATA_MIGRATE" == "false" ];
+  then
+    echo "Running data schema load…"
+    bundle exec rake data:schema:load
```

Comment:
Only if it's explicitly set to `false` do we do the unusual route of only loading the schema. All other times the default behaviour should be the previous `data:migrate` as we trust the state of the schema to manage what migrations need to be run.